api/src/main/java/com/example/api/security/SecurityConfig.java

```diff
@@ -27,7 +27,6 @@ public class SecurityConfig extends WebSecurityConfigurerAdapter {
     protected void configure(HttpSecurity security) throws Exception {
```

Comment:
Nie ma ustawionego CORS - będzie się sypać wszędzie poza Postmanem (sprawdzone), trzeba dodać
```suggestion
    protected void configure(HttpSecurity security) throws Exception {
    security.cors().configurationSource(request -> new CorsConfiguration(). applyPermitDefaultValues());
```

To ustawi permit all na każdy CORS (a przynajmniej tak to wygląda jak odpalam miniprzykład w React).

Reply:
Już zaraz to naprawię. Totalnie zapomniałem o CORS.